src/main/java/core/basesyntax/model/User.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.model;
+
+import java.util.List;
+
+public class User {
+    private int id;
```

Comment:
in the Future all of our entities will have id's, but now let's remove it

Reply:
Ok, it's just a habit)